README.md

```diff
@@ -1,13 +1,27 @@
-# lium - Abstraction Layer of ChromiumOS development
+# lium - Make ChromiumOS development extreamely easy
 
-`lium` is an abstruction layer of ChromiumOS development environment and workflows.
+`lium` is an extreamely user-friendly tool for ChromiumOS developers.
 
-It provides a simple interface for common tasks for ChromiumOS developers,
-with plenty of automatic error recovery mechanisms to avoid keep sticking your eyes on the display(s),
-give you some time for a nap and/or coffee, or other tasks ;)
+It provides a simple way to do common development tasks and make the bareer for contributing to ChromiumOS and lium itself low as much as possible.
+
+It also comes with plenty of automatic error recovery mechanisms to avoid keep sticking your eyes on displays.
 
 Also, it manages local development hardware including DUTs and Servos, and act as working examples of commands to interact with them.
 
+We hope lium gives you some time for a nap and/or coffee, or other tasks by making your work more effective ;)
+
+## Core principles
+
+- P0: Make the basic ChromiumOS development workflow extremely easy as like as a usual software on GitHub
+  - Background: There have been a high barrier to get started with the ChromiumOS development. It terrifies new comers sometimes, and such environment is not sustainable nor efficient. This tool aims to solve such problems at a top priority.
```

Comment:
Have been huge barriers
Newcomers
Is not sustainable or efficient
The top priority of lium is to mitigate this high barrier. 

Idk why but this just sounds better to me. It softens it

Reply:
JFYI: P0 is a Googleism, I would spell it out as top priority.

Reply:
I think P0 term is now commonly used so I would like to keep it as is.
updated the text based on Brandon's suggestion

Reply:
> P0 – the site is down and all work stops, until this issue is resolved.

https://www.linkedin.com/pulse/definition-priority-codes-p0-p1-p2-p3-p4-technical-victor-pudeyev

Also: https://en.m.wikipedia.org/wiki/P0

has nothing.